haskellPackages: update stackage and hackage

maralorn opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. You may be able to find an up-to-date Hydra build report at [cdepillabout/nix-haskell-updates-status](https://github.com/cdepillabout/nix-haskell-updates-status).

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/H6migHmKX7xHoTFa) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #192273. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

maralorn committed:
haskellPackages: stackage LTS 19.24 -> LTS 19.25

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

maralorn committed:
all-cabal-hashes: 2022-09-21T15:25:15Z -> 2022-09-25T05:09:53Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

maralorn committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

maralorn commented:
@domenkozar I haven‘t looked much into it, but there is a cachix build error going on: https://hydra.nixos.org/build/192123042/nixlog/3

domenkozar committed:
haskellPackages.cachix: fix build

domenkozar commented:
@maralorn thanks, pushed a fix.

cdepillabout committed:
haskellPackages.polysemy-zoo: unbreak

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1782156](https://hydra.nixos.org/eval/1782156) of nixpkgs commit [575babf](https://github.com/NixOS/nixpkgs/commits/575babff621cecd2df750f7941b59823d0098f81) as of 2022-09-26 13:36 UTC*

:red_circle: **Branch not mergeable**
  * `mergeable` jobset failed.
  * `maintained` jobset failed.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1782156?filter=.aarch64-linux) | 32 | 31 | 9 | 6615 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1782156?filter=.x86_64-darwin) | 71 | 25 |  | 6533 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1782156?filter=.x86_64-linux) | 9 | 8 | 3 | 6708 | 
#### Maintained packages with build failure
- [ ] [ghcHEAD](https://hydra.nixos.org/eval/1782156?filter=ghcHEAD) @sternenseemann 
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191933390) [[:apple::x:]](https://hydra.nixos.org/build/191933386) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191933392) [haskell.compiler](https://hydra.nixos.org/eval/1782156?filter=haskell.compiler.ghcHEAD)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191933388) [[:apple::x:]](https://hydra.nixos.org/build/191933391) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191933387) [haskell.compiler.native-bignum](https://hydra.nixos.org/eval/1782156?filter=haskell.compiler.native-bignum.ghcHEAD)
  -   [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191933385) [pkgsMusl.haskell.compiler](https://hydra.nixos.org/eval/1782156?filter=pkgsMusl.haskell.compiler.ghcHEAD)
  -   [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191933393) [pkgsMusl.haskell.compiler.native-bignum](https://hydra.nixos.org/eval/1782156?filter=pkgsMusl.haskell.compiler.native-bignum.ghcHEAD)
#### Maintained packages with failed dependency
- [ ] [haskell-language-server](https://hydra.nixos.org/eval/1782156?filter=haskell-language-server) @maralorn
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123378) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123726) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123529) [toplevel](https://hydra.nixos.org/eval/1782156?filter=haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192122991) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123316) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123677) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc8107.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123441) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123420) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123330) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc884.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192122990) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123166) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123575) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc902.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123609) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123193) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123185) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc924.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123457) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123026) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123477) [haskellPackages](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.haskell-language-server)
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192220118) [maintained](https://hydra.nixos.org/eval/1782156?filter=maintained) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192220131) [mergeable](https://hydra.nixos.org/eval/1782156?filter=mergeable) @cdepillabout @expipiplus1 @maralorn @sternenseemann
#### Unmaintained packages with build failure
<details><summary>90 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123508) [[:apple::x:]](https://hydra.nixos.org/build/192123160) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123281) [haskellPackages.th-utilities](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.th-utilities)  :arrow_heading_up: 6 | 47
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191867082) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865210) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191871098) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 5 | 36
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191864684) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191864265) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868477) [haskellPackages.hw-json-simd](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-json-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191866772) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191870439) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191866535) [haskellPackages.hw-simd](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123290) [[:apple::x:]](https://hydra.nixos.org/build/192122983) [[:penguin::x:]](https://hydra.nixos.org/build/192123493) [haskellPackages.sint](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.sint)  :arrow_heading_up: 2 | 6
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426554) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190433701) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435927) [haskellPackages.long-double](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.long-double)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123090) [[:apple::x:]](https://hydra.nixos.org/build/192123346) [[:penguin::x:]](https://hydra.nixos.org/build/192123086) [haskellPackages.portray-pretty](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.portray-pretty)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191871235) [[:apple::x:]](https://hydra.nixos.org/build/191870056) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868195) [haskellPackages.quic](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.quic)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123230) [[:apple::x:]](https://hydra.nixos.org/build/192122979) [[:penguin::x:]](https://hydra.nixos.org/build/192123345) [haskellPackages.ten](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ten)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191868147) [[:apple::x:]](https://hydra.nixos.org/build/191865112) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191871356) [haskellPackages.junit-xml](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.junit-xml)  :arrow_heading_up: 1 | 9
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190418721) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190424366) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190418471) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.freetype2)  :arrow_heading_up: 1 | 8
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123375) [[:apple::x:]](https://hydra.nixos.org/build/192123692) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123417) [haskellPackages.binary-serialise-cbor](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.binary-serialise-cbor)  :arrow_heading_up: 1 | 3
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426179) [[:apple::x:]](https://hydra.nixos.org/build/190428059) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423250) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123080) [[:apple::x:]](https://hydra.nixos.org/build/192123187) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123476) [haskellPackages.hls-alternate-number-format-plugin](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hls-alternate-number-format-plugin)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123510) [[:apple::x:]](https://hydra.nixos.org/build/192123181) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123146) [haskellPackages.hls-call-hierarchy-plugin](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hls-call-hierarchy-plugin)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123430) [[:apple::x:]](https://hydra.nixos.org/build/192123152) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123411) [haskellPackages.hls-code-range-plugin](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hls-code-range-plugin)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191867640) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191871233) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191864023) [haskellPackages.kazura-queue](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.kazura-queue)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190420884) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190436438) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190425960) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191237050) [[:apple::x:]](https://hydra.nixos.org/build/190430196) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191237036) [haskellPackages.openal-ffi](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.openal-ffi)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190426560) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190422117) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422325) [haskellPackages.swisstable](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.swisstable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190434587) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427671) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433104) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191867865) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865502) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870778) [haskellPackages.flatparse](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.flatparse)  :arrow_heading_up: 0 | 14
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428795) [[:apple::x:]](https://hydra.nixos.org/build/190430540) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190429741) [haskellPackages.PyF](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.PyF)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435452) [[:apple::x:]](https://hydra.nixos.org/build/190423209) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423465) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191869227) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191867180) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191864716) [haskellPackages.json-rpc](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.json-rpc)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191868965) [[:apple::x:]](https://hydra.nixos.org/build/191869630) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870717) [haskellPackages.posix-socket](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.posix-socket)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191863336) [[:apple::x:]](https://hydra.nixos.org/build/191870633) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191866233) [haskellPackages.gi-gdkx11](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gi-gdkx11)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190420703) [[:apple::x:]](https://hydra.nixos.org/build/190437843) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434958) [haskellPackages.hamid](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hamid)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190427217) [[:apple::x:]](https://hydra.nixos.org/build/190422876) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190434364) [haskellPackages.hmatrix-morpheus](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hmatrix-morpheus)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435315) [[:apple::x:]](https://hydra.nixos.org/build/190424984) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422869) [haskellPackages.huckleberry](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.huckleberry)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190422851) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190421788) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190427574) [haskellPackages.picosat](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190421878) [[:apple::x:]](https://hydra.nixos.org/build/190420004) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419766) [haskellPackages.select](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.select)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191863402) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866594) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868212) [haskellPackages.simple-vec3](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.simple-vec3)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190435304) [[:apple::x:]](https://hydra.nixos.org/build/190437845) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190425782) [haskellPackages.sysinfo](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.sysinfo)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190426515) [[:apple::x:]](https://hydra.nixos.org/build/190430439) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423606) [haskellPackages.FractalArt](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.FractalArt) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190435051) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434382) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436557) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123608) [[:apple::x:]](https://hydra.nixos.org/build/192123121) [[:penguin::x:]](https://hydra.nixos.org/build/192123698) [haskellPackages.brassica](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.brassica) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429501) [[:apple::x:]](https://hydra.nixos.org/build/190434413) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436327) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191865311) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866048) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868495) [haskellPackages.comfort-fftw](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.comfort-fftw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436349) [[:apple::x:]](https://hydra.nixos.org/build/190429906) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437140) [haskellPackages.diskhash](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.diskhash) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191867973) [[:apple::x:]](https://hydra.nixos.org/build/191866617) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191866484) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190422331) [[:apple::x:]](https://hydra.nixos.org/build/190421828) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421042) [haskellPackages.fudgets](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.fudgets) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123180) [[:apple::x:]](https://hydra.nixos.org/build/192123334) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123504) [haskellPackages.genvalidity-hspec-optics](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.genvalidity-hspec-optics) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191867135) [[:apple::x:]](https://hydra.nixos.org/build/191867433) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865428) [haskellPackages.gerrit](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gerrit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428783) [[:apple::x:]](https://hydra.nixos.org/build/190418276) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435006) [haskellPackages.ghc-gc-hook](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ghc-gc-hook) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/191867187) [haskellPackages.gi-gtkosxapplication](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gi-gtkosxapplication) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191177999) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191177946) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gnome-keyring) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190425865) [haskellPackages.gtk-mac-integration](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gtk-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191178016) [[:apple::x:]](https://hydra.nixos.org/build/190424726) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191177958) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190433684) [haskellPackages.gtk3-mac-integration](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.gtk3-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123301) [[:apple::x:]](https://hydra.nixos.org/build/192123215) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123578) [haskellPackages.hamilton](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hamilton) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190428754) [[:apple::x:]](https://hydra.nixos.org/build/190424290) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190420486) [haskellPackages.hid](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191868568) [[:apple::x:]](https://hydra.nixos.org/build/191867979) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870760) [haskellPackages.highlight](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.highlight) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123289) [[:apple::x:]](https://hydra.nixos.org/build/192123174) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123509) [haskellPackages.hls-brittany-plugin](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hls-brittany-plugin) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190437030) [[:apple::x:]](https://hydra.nixos.org/build/190420833) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435871) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190438505) [[:apple::x:]](https://hydra.nixos.org/build/190422004) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435562) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190432750) [[:apple::x:]](https://hydra.nixos.org/build/190419419) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190433284) [haskellPackages.interprocess](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.interprocess) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191177923) [[:apple::x:]](https://hydra.nixos.org/build/190427460) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191177934) [haskellPackages.intricacy](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.intricacy) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191187639) [[:apple::x:]](https://hydra.nixos.org/build/191187804) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191187796) [haskellPackages.ipcvar](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ipcvar) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190429809) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190420867) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190435388) [haskellPackages.jammittools](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.jammittools) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/190429773) [haskellPackages.kqueue](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.kqueue) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190429726) [[:apple::x:]](https://hydra.nixos.org/build/190421244) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190422406) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123667) [[:apple::x:]](https://hydra.nixos.org/build/192123623) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123096) [haskellPackages.massiv-test](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.massiv-test) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191863791) [[:apple::x:]](https://hydra.nixos.org/build/191865919) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870868) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190437330) [[:apple::x:]](https://hydra.nixos.org/build/190420114) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424042) [haskellPackages.memfd](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.memfd) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190431691) [[:apple::x:]](https://hydra.nixos.org/build/190420349) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190436127) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190438583) [[:apple::x:]](https://hydra.nixos.org/build/190431381) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419732) [haskellPackages.nano-cryptr](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.nano-cryptr) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123684) [[:apple::x:]](https://hydra.nixos.org/build/192123629) [[:penguin::x:]](https://hydra.nixos.org/build/192123427) [haskellPackages.nix-serve-ng](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.nix-serve-ng) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191866934) [[:apple::x:]](https://hydra.nixos.org/build/191864891) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865599) [haskellPackages.persistent-pagination](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.persistent-pagination) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123264) [[:apple::x:]](https://hydra.nixos.org/build/192123657) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123358) [haskellPackages.phatsort](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.phatsort) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191865907) [[:apple::x:]](https://hydra.nixos.org/build/191864423) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865192) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123704) [[:apple::x:]](https://hydra.nixos.org/build/192123537) [[:penguin::x:]](https://hydra.nixos.org/build/192123173) [haskellPackages.portray-prettyprinter](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.portray-prettyprinter) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419667) [[:apple::x:]](https://hydra.nixos.org/build/190428126) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424911) [haskellPackages.posix-timer](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.posix-timer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190419559) [[:apple::x:]](https://hydra.nixos.org/build/190418650) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437706) [haskellPackages.procex](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.procex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191871269) [[:apple::x:]](https://hydra.nixos.org/build/191866561) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865995) [haskellPackages.pthread](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.pthread) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123267) [[:apple::x:]](https://hydra.nixos.org/build/192123588) [[:penguin::x:]](https://hydra.nixos.org/build/192123102) [haskellPackages.rec-def](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.rec-def) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190427344) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428826) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438377) [haskellPackages.risc386](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.risc386) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123505) [[:apple::x:]](https://hydra.nixos.org/build/192123157) [[:penguin::x:]](https://hydra.nixos.org/build/192122995) [haskellPackages.rle](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.rle) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191237009) [[:apple::x:]](https://hydra.nixos.org/build/190428959) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191237020) [haskellPackages.sfml-audio](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.sfml-audio) 
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191868784) [[:apple::x:]](https://hydra.nixos.org/build/191871109) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191869724) [haskellPackages.skews](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.skews) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/191186688) [[:apple::x:]](https://hydra.nixos.org/build/191187386) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191188438) [haskellPackages.slugify](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.slugify) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191869673) [[:apple::x:]](https://hydra.nixos.org/build/191868286) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870350) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/192123280) [[:apple::x:]](https://hydra.nixos.org/build/192123564) [[:penguin::x:]](https://hydra.nixos.org/build/192123435) [haskellPackages.toml-reader-parse](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.toml-reader-parse) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190421659) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190430013) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190419700) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/190429268) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190428038) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190426739) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191188559) [[:apple::x:]](https://hydra.nixos.org/build/191187837) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191188435) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123343) [[:apple::x:]](https://hydra.nixos.org/build/192123313) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123502) [haskellPackages.yesod-page-cursor](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.yesod-page-cursor) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190426980) [[:apple::x:]](https://hydra.nixos.org/build/190426061) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190420502) [haskellPackages.yoga](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.yoga) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190436342) [[:apple::x:]](https://hydra.nixos.org/build/190436771) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190438349) [haskellPackages.zot](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.zot) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/190418762) [[:apple::x:]](https://hydra.nixos.org/build/190433345) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190423003) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>51 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123269) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123395) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123566) [haskellPackages.store](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.store)  :arrow_heading_up: 4 | 42
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191863886) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866262) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191869519) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 4 | 35
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191868340) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191871049) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865822) [haskellPackages.BiobaseTypes](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.BiobaseTypes)  :arrow_heading_up: 3 | 21
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191870836) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865996) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868806) [haskellPackages.hw-json-standard-cursor](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-json-standard-cursor)  :arrow_heading_up: 2 | 6
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191866300) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865835) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191866072) [haskellPackages.hw-json-simple-cursor](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-json-simple-cursor)  :arrow_heading_up: 2 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191865191) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191869375) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870109) [haskellPackages.BiobaseENA](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.BiobaseENA)  :arrow_heading_up: 1 | 18
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123300) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123731) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192122993) [haskellPackages.Color](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.Color)  :arrow_heading_up: 1 | 7
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123461) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123163) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123198) [haskellPackages.fin-int](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.fin-int)  :arrow_heading_up: 1 | 5
- [ ] [hoogle](https://hydra.nixos.org/eval/1782156?filter=hoogle)  :arrow_heading_up: 1 | 3
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191870637) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191864831) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868852) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc8107.hoogle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191870155) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865261) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865678) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc884.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191870885) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865524) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865041) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc902.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191864310) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191868293) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191864580) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1782156?filter=haskell.packages.ghc924.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191863915) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191869115) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191869356) [haskellPackages](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hoogle)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191870792) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866103) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191870248) [haskellPackages.hw-json](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-json)  :arrow_heading_up: 1 | 3
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192122989) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123328) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123708) [haskellPackages.cursor-gen](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.cursor-gen)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191867572) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191867961) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865682) [haskellPackages.http3](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.http3)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191870098) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191866554) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191866164) [haskellPackages.wss-client](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.wss-client)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191864949) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191868620) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191866508) [haskellPackages.BiobaseXNA](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.BiobaseXNA)  :arrow_heading_up: 0 | 17
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191870142) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191865356) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865674) [haskellPackages.pretty-diff](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.pretty-diff)  :arrow_heading_up: 0 | 12
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191863986) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191863825) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191863296) [haskellPackages.BiobaseFasta](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.BiobaseFasta)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191868845) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191871355) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191863432) [haskellPackages.hw-dsv](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-dsv)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123694) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123191) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123052) [haskellPackages.fresco-binding](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.fresco-binding)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191865438) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191867073) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191865386) [haskellPackages.hw-json-lens](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-json-lens)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123228) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123247) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123607) [haskellPackages.massiv-io](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.massiv-io)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191869124) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191867537) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191867810) [haskellPackages.HMarkov](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.HMarkov) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191866622) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865655) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191869570) [haskellPackages.align-audio](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.align-audio) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123524) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123194) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123188) [haskellPackages.cursor-fuzzy-time-gen](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.cursor-fuzzy-time-gen) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123149) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123371) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192122996) [haskellPackages.dependent-literals](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.dependent-literals) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190434920) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190437763) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190437884) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190428304) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190426306) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190428625) [haskellPackages.harfbuzz-pure](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.harfbuzz-pure) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191863246) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191870750) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191867857) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191869166) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866681) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191867352) [haskellPackages.hriemann](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hriemann) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190432249) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190427429) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190421622) [haskellPackages.hs-swisstable-hashtables-class](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hs-swisstable-hashtables-class) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191871354) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191868101) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191868323) [haskellPackages.hw-simd-cli](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.hw-simd-cli) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190426859) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190434719) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190432027) [haskellPackages.kmn-programming](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.kmn-programming) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123270) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123424) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123432) [haskellPackages.libmdbx](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.libmdbx) 
- [ ] [[:iphone::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191866199) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191865887) [[:penguin::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/191870072) [haskellPackages.network-messagepack-rpc-websocket](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.network-messagepack-rpc-websocket) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123212) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123519) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123131) [haskellPackages.portray-diff-hunit](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.portray-diff-hunit) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123159) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123572) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123703) [haskellPackages.portray-diff-quickcheck](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.portray-diff-quickcheck) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123013) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/192123060) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123332) [haskellPackages.rounded](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191866551) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191865116) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191863437) [haskellPackages.rounded-hw](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.rounded-hw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123712) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123613) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123040) [haskellPackages.safe-money-store](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.safe-money-store) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191866227) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/191866492) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191869426) [haskellPackages.sound-collage](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.sound-collage) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/192123109) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123536) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/192123642) [haskellPackages.store-streaming](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.store-streaming) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191866956) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191870656) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191867458) [haskellPackages.tasty-test-reporter](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.tasty-test-reporter) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123061) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123691) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123182) [haskellPackages.ten-lens](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ten-lens) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123150) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123571) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/192123663) [haskellPackages.ten-unordered-containers](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.ten-unordered-containers) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/190432799) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/190425240) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/190424340) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191869991) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191869862) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191867416) [haskellPackages.warp-quic](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.warp-quic) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/191188144) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/191188149) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/191186835) [haskellPackages.xbattbar](https://hydra.nixos.org/eval/1782156?filter=haskellPackages.xbattbar) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[amazonka-core](https://packdeps.haskellers.com/reverse/amazonka-core) :arrow_heading_up: 185  
[gogol-core](https://packdeps.haskellers.com/reverse/gogol-core) :arrow_heading_up: 184  
[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[util](https://packdeps.haskellers.com/reverse/util) :arrow_heading_up: 49  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[amazonka](https://packdeps.haskellers.com/reverse/amazonka) :arrow_heading_up: 43  
[accelerate](https://packdeps.haskellers.com/reverse/accelerate) :arrow_heading_up: 42  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 42  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 33  
[rank1dynamic](https://packdeps.haskellers.com/reverse/rank1dynamic) :arrow_heading_up: 33  
[distributed-static](https://packdeps.haskellers.com/reverse/distributed-static) :arrow_heading_up: 31  
[language-ecmascript](https://packdeps.haskellers.com/reverse/language-ecmascript) :arrow_heading_up: 31  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[mmsyn3](https://packdeps.haskellers.com/reverse/mmsyn3) :arrow_heading_up: 28  
[autodocodec-yaml](https://packdeps.haskellers.com/reverse/autodocodec-yaml) :arrow_heading_up: 27  
[crypto-numbers](https://packdeps.haskellers.com/reverse/crypto-numbers) :arrow_heading_up: 25  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[sydtest](https://packdeps.haskellers.com/reverse/sydtest) :arrow_heading_up: 24  
[crypto-pubkey](https://packdeps.haskellers.com/reverse/crypto-pubkey) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[alg](https://packdeps.haskellers.com/reverse/alg) :arrow_heading_up: 21  
[amazonka-s3](https://packdeps.haskellers.com/reverse/amazonka-s3) :arrow_heading_up: 21  
[mmsyn2](https://packdeps.haskellers.com/reverse/mmsyn2) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[fay](https://packdeps.haskellers.com/reverse/fay) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
[gtksourceview2](https://packdeps.haskellers.com/reverse/gtksourceview2) :arrow_heading_up: 18  
[ukrainian-phonetics-basic](https://packdeps.haskellers.com/reverse/ukrainian-phonetics-basic) :arrow_heading_up: 18  
[HGamer3D-Data](https://packdeps.haskellers.com/reverse/HGamer3D-Data) :arrow_heading_up: 17  
[certificate](https://packdeps.haskellers.com/reverse/certificate) :arrow_heading_up: 17  
[dbus-client](https://packdeps.haskellers.com/reverse/dbus-client) :arrow_heading_up: 17  
[gconf](https://packdeps.haskellers.com/reverse/gconf) :arrow_heading_up: 17  
[gtk-serialized-event](https://packdeps.haskellers.com/reverse/gtk-serialized-event) :arrow_heading_up: 17  
[cuda](https://packdeps.haskellers.com/reverse/cuda) :arrow_heading_up: 16  
[happstack-jmacro](https://packdeps.haskellers.com/reverse/happstack-jmacro) :arrow_heading_up: 16  
[manatee-core](https://packdeps.haskellers.com/reverse/manatee-core) :arrow_heading_up: 16  
[monads-fd](https://packdeps.haskellers.com/reverse/monads-fd) :arrow_heading_up: 16  
[tls-extra](https://packdeps.haskellers.com/reverse/tls-extra) :arrow_heading_up: 16  
[ADPfusion](https://packdeps.haskellers.com/reverse/ADPfusion) :arrow_heading_up: 15  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

sternenseemann committed:
haskell.compiler: upgrade to 9.2.4 for 9.2.* binary compiler

cdepillabout commented:
It looks like the Haskell language server failure on Darwin just needed a restart of the Hydra job.  It looks like it is now passing.

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh

sternenseemann committed:
haskellPackages.hs-mesos: remove stale override

pkgs.mesos was finally removed and the package hasn't been able to be
built for a while.

sternenseemann committed:
haskellPackages.nix-serve-ng: fix build

* Workaround missing files in sdist
* Add missing undeclared boost dependency

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
haskellPackages.disco: run offline tests only

sternenseemann committed:
haskellPackages.servant-polysemy: mark as broken

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates

Artturin commented on pkgs/development/haskell-modules/configuration-nix.nix:
```diff
@@ -154,11 +154,18 @@ self: super: builtins.intersectAttrs super {
   # Add necessary reference to gtk3 package
   gi-dbusmenugtk3 = addPkgconfigDepend pkgs.gtk3 super.gi-dbusmenugtk3;
 
-  hs-mesos = overrideCabal (drv: {
-    # Pass _only_ mesos; the correct protobuf is propagated.
-    extraLibraries = [ pkgs.mesos ];
-    preConfigure = "sed -i -e /extra-lib-dirs/d -e 's|, /usr/include, /usr/local/include/mesos||' hs-mesos.cabal";
-  }) super.hs-mesos;
```
@sternenseemann you forgot to remove https://github.com/NixOS/nixpkgs/blob/6b2d239024f405d60d98d12f8f31dcfd42ee6a94/pkgs/top-level/packages-config.nix#L37
